proposals/aip-1.md

```diff
@@ -0,0 +1,128 @@
+
+The [@Agorex-io/proposal-editors](https://github.com/orgs/Agorex-io/teams/proposal-editors) facilitate champions and authors in the process. If they recommend it, a Proposal Editor will assign the AIP a number (generally the issue or PR number related to the AIP), label it as Informational or Process, give it status **Draft** (this is all done in the same PR branch). The AIP editor will not unreasonably deny an AIP in Draft status. Reasons for denying proposal status include duplication of effort, being technically unsound, not providing proper rational, or not in keeping with the Agorex values.
+
+For an AIP to be merged Draft status it still must meet certain minimum criteria. It must be a clear and complete description of the proposed process or issue. The process or issue must represent a net improvement.
+
+## Acceptance Process ##
+
+Once an AIP has been reviewed by a Proposal Editor, the Editor can recommend it for a vote for approval by the [Agorex Board of Directors](https://github.com/orgs/Agorex-io/teams/board-of-directors). Proposal pull requests must be reviewed and approved by a majority of the [Agorex Board of Directors](https://github.com/orgs/Agorex-io/teams/board-of-directors). We use GitHub's [required reviews for pull requests](https://help.github.com/articles/about-required-reviews-for-pull-requests/) to enforce this review and approval process. 
+
+
+While the [Agorex Board of Directors](https://github.com/orgs/Agorex-io/teams/board-of-directors) should quite liberally accept Draft status proposals subject to the criteria above, merging a proposal with a **Final** status such as **Final** or **Replaced** requires more review and rigor since, by merging a Final proposal The Board is agreeing to execute the proposed process on behalf of the community.
+
+Once the proposal is reviewed and approved by The Board on behalf of the community, the status will be changed to "Final" and it will be merged.
+
+
+## Deferred, Rejected, and Replaced ##
+
+An AIP can also be assigned status **Deferred**. The AIP author or editor can assign the AIP this status when no progress is being made on the AIP. Once an AIP is deferred, the AIP editor can re-assign it to Draft status.
+
+An AIP can also be **Rejected**. Perhaps after all is said and done it was not a good idea. It is still important to have a record of this fact.
+
+AIPs can also be superseded by a different AIP, rendering the original obsolete. In this the superseeded Proposal should be marked as **Replaced** as part of the PR that the superseeding proposal is being proposed in.
+
+## Proposal Statuses ##
+The possible paths of the status of AIPs are as follows:
+
+  Draft -> Accepted -> Final
+  Draft -> Withdrawn
+  Draft -> Rejected
+  Draft -> Deferred
+  Final -> Replaced
```

Comment:
EIPs (and the python enhancement proposal that it is derived from) leave them so that people can see the past decision that was rejected for future reference. I think that makes sense.

Reply:
Okay, so we will need labels for these different states so that we can filter them out.

Reply:
Yes, the status is in the preamble.